langchain4j-core/src/main/java/dev/langchain4j/data/document/Metadata.java

```diff
@@ -124,6 +139,30 @@ public String getString(String key) {
             "It cannot be returned as a String.", key, value, value.getClass().getName());
     }
 
+    /**
+     * Returns the collection of {@code String} values associated with the given key.
+     *
+     * @param key the key
+     * @return the collection of {@code String} values associated with the given key, or an empty collection if the key is not present.
+     * @throws RuntimeException if the value is not of type Collection
+     */
+    public Collection<String> getStrings(String key) {
+        if (!containsKey(key)) {
+            return Collections.emptyList();
+        }
+
+        Object values = metadata.get(key);
+        if (values instanceof Collection<?>) {
+            return ((Collection<?>) values).stream()
+                    .filter(String.class::isInstance)
```

Comment:
Should we throw an exception if the collection contains a value that isn't a String here, instead of silently filtering it away?

Reply:
I chose the filtering approach to allow collections of mixed types.

Reply:
Does any store support collections of mixed types?